code-challenges.test.js

```diff
@@ -15,43 +15,144 @@
 // a) Create a test with an expect statement using the variable provided.
 // HINT: Check out this resource: https://jestjs.io/docs/expect#expectarraycontainingarray
 
-const colors1 = ["purple", "blue", "green", "yellow", "pink"]
+// describe("shuffleContent", () => {
+//   it("removes the first item from the array and shuffles the remaining content", () => {
+//     const colors1 = ["purple", "blue", "green", "yellow", "pink"]
+//     const colors2 = [
+//       "chartreuse",
+//       "indigo",
+//       "periwinkle",
+//       "ochre",
+//       "aquamarine",
+//       "saffron"
+//     ]
+//     expect(shuffleContent(colors1)).not.toEqual(expect.arrayContaining(expected))
+//     expect(shuffleContent(colors2)).not.toEqual(expect.arrayContaining(expected))
```

Comment:
Good job finding the arrayContaining to use in the test. However, you are telling the test not to expect an array containing "expected".
- `expected` is a variable that is not defined, so the test will not know what you're talking about
- If you put the expected output where `expected` is, the test will check if the output does not have those items in the array that is returned from the function, which is not what you want!